ovirt_imageio/client/_options.py

```diff
@@ -190,7 +190,7 @@ def __init__(self):
             version=f'%(prog)s {version.string}')
         self._commands = self._parser.add_subparsers(title="commands")
 
-    def add_sub_command(self, name, help, func, transfer_options=True):
+    def add_sub_command(self, name, help = "help", func = lambda x: None, transfer_options=True):
```

Comment:
Please remove the spaces surrounding the `=` symbol, following [PEP-0008](https://peps.python.org/pep-0008/#other-recommendations) recommendation (and also for matching surrounding code).